.github/dependabot.yml

```diff
@@ -0,0 +1,30 @@
+# Basic dependabot.yml file with minimum configuration for gomod.
+
+version: 2
+updates:
+  # Enable version updates for netmaker
+  - package-ecosystem: "gomod"
+    directory: "/"
+    # Check for updates every day (weekdays)
+    schedule:
+      interval: "weekly"
+    target-branch: "develop"
+  # Enable version updates for netclient
+  - package-ecosystem: "gomod"
```

Comment:
Will the bot raise a PR? Because I'm pretty sure we'll not want to jump on every latest dependency version

Reply:
yes, look at the PR in netmaker repo 1817, 1818, 1819 for example